cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -134,8 +134,6 @@ if (EMSCRIPTEN_FORCE_COMPILERS)
 	set(CMAKE_CXX_COMPILER_ID Clang)
 	set(CMAKE_CXX_STANDARD_COMPUTED_DEFAULT 98)
 
-	set(CMAKE_C_COMPILER_TARGET "asmjs-unknown-emscripten")
```

Comment:
Hm, OK. In the fullness of time I actually do think it makes sense to have this in the CMake description file because that would allow use of a system-installed clang that supports native or wasm. But that would be after triple convergence, and maybe would be  different CMake file than this one anyway, so I'm OK dropping this now, if @kripken and/or @juj  are.